external/fv3fit/tests/training/test_reservoir.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
+from fv3fit.tfdataset import tfdataset_from_batches
+from fv3fit.reservoir.domain import concat_variables_along_feature_dim
+from fv3fit.reservoir.train import train_reservoir_model
+from fv3fit.reservoir.config import (
+    ReservoirTrainingConfig,
+    CubedsphereSubdomainConfig,
+    ReservoirHyperparameters,
+    BatchLinearRegressorHyperparameters,
+)
+from tests.training.test_train import (
+    get_dataset_default,
+    get_uniform_sample_func,
+)
+
+
+def test_get_datasets():  # nx, ny, n_feature, n_sample):
+    n_sample = 10
+    n_tile, nx, ny, nz = 1, 12, 12, 5
```

Comment:
Is this training intended to only apply for single tiles or is it expected to generalize to the multi-tile case?